extensions/helpops.c

```diff
@@ -53,9 +53,9 @@ mo_dehelper(struct MsgBuf *msgbuf_p, struct Client *client_p, struct Client *sou
 {
 	struct Client *target_p;
 
-	if (!IsOperAdmin(source_p))
+	if (!HasPrivilege(source_p, "oper:dehelper"))
```

Comment:
Likewise.

Reply:
These are both because they're provided only by extensions, but I guess I'll add a section for them.